project-base/tests/ShopBundle/Test/ParameterTransactionFunctionalTestCase.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ShopBundle\Test;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterFacade;
+
+class ParameterTransactionFunctionalTestCase extends TransactionFunctionalTestCase
+{
+    /**
+     * @param string $parameterValueNameId
+     * @return int
+     */
+    protected function getParameterValueId(string $parameterValueNameId): int
```

Comment:
Maybe rename it to getParameterValueIdForFirstDomain? because I think it does that.

Reply:
renamed in f779df457